_mobile/ios.md

```diff
@@ -194,6 +193,37 @@ In the build settings, search for **other linker flags**.  Add a custom linker f
 
 Currently, the iOS framework uses the Pytorch C++ front-end APIs directly. The C++ document can be found here https://pytorch.org/cppdocs/. To learn more about it, we recommend exploring the [C++ front-end tutorials](https://pytorch.org/tutorials/advanced/cpp_frontend.html) on PyTorch webpage. In the meantime, we're working on providing the Swift/Objective-C API wrappers to PyTorch.
 
+
+### Custom build
+
+Starting from 1.4.0, PyTorch supports custom build. You can now build the PyTorch library that only contains the operators needed by your model. To do that, follow the steps below
+
+1. Verify your PyTorch version is 1.4.0 or above. You can do that by checking the value of `torch.__version__`.
+2. To dump the operators in your model, we need to run a few lines of Python code
+
+    ```python
+    import torch, yaml
+    model = torch.jit.load("example.pt")
+    ops = torch.jit.export_opnames(m)
+    f = open('example.yaml', 'w')
+    yaml.dump(ops, f)
+    ```
+The `export_opnames` API is available in `torch.jit` module, and we need to save the result to a yaml file.
```

Comment:
- That's helpful to call out where it is available; could you elaborate a bit more about what export_opnames is doing? Also, will there be someplace in the API docs we can link to about the export_opnames API (such as the docstrings https://github.com/pytorch/pytorch/blob/master/torch/jit/__init__.py#L243-L247)? Could provide more context to the reader. 

- Suggested rephrase:
"In the snippet above, you first need to load the ScriptModule. Then, use export_opnames to return a list of operator names of the ScriptModule and its submodules. Lastly, save the result in a yaml file."

Reply:
Unfortunately, I didn't find a doc about export_opnames. I guess that is a new feature in 1.4.0 for JIT as well. So the docs haven't been updated yet.